zebra-script/src/lib.rs

```diff
@@ -167,39 +173,33 @@ mod tests {
             .branch_id()
             .expect("Blossom has a ConsensusBranchId");
 
-        is_valid(transaction, branch_id, (input_index, output))?;
+        let verifier = super::PrecomputedTransaction::new(&transaction);
```

Comment:
Can we also have a separate test that re-uses a pre-computed script?

I'd like a separate test because single-use and multi-use have different semantics.